package.json

```diff
@@ -17,9 +17,13 @@
   },
   "dependencies": {
     "code-push": "1.5.2-beta",
-    "semver": "^5.1.0"
+    "semver": "^5.1.0",
+    "rnpm": "1.4.1"
   },
-  "devDependencies": {
+  "peerDependencies": {
     "react-native": "0.19.0"
```

Comment:
Should this be >=0.19.0? We don't want to artificially block future versions that might just work.